Configuring middleware in the component (#3937)
dapr/docs: daprdocs/content/en/developing-applications/develop-components/middleware-components/develop-middleware.md

---
type: docs

> title: "How to: Author middleware components"

Reviewer: ```suggestion
title: "How to: Implement middleware components"
```

> linkTitle: "How to: Author middleware"

Reviewer: ```suggestion
linkTitle: "How to: Implement middleware components"
```

weight: 100

> description: "Learn how to develop middleware components"

Reviewer: ```suggestion
description: "Learn how to author and implement middleware components"
```

---

> Dapr allows custom processing pipelines to be defined by chaining a series of middleware components. In this guide, you'll learn how to create a middleware component. To learn how to configure an existing middleware component, see [Configure middleware components]({{< ref middleware.md >}})

Reviewer: ```suggestion
Dapr allows custom processing pipelines to be defined by chaining a series of middleware components. In this guide, you'll learn how to create a middleware component. To learn how to use a middleware component, see [Configure middleware components]({{< ref middleware.md >}})
```
